samples/minimum_criteria.rst

```diff
@@ -0,0 +1,20 @@
+.. _definition_and_minimum_criteria:
+
+Definition & Minimum Criteria
```

Comment:
I find this definition very thin in content and is not aadding much beyond what is already known. 
Split the title and add some structure and more content and guideline re samples